src/ui/screens/AccountsScreen.tsx

```diff
@@ -43,32 +43,67 @@ export function AccountsScreen({navigation}: Props) {
     return [...acc, {identity: current, account}];
   }, []);
 
-  const [sortBy, setSortBy] = React.useState<'name' | 'favorites'>('name');
+  const [sortBy, setSortBy] = React.useState<SortBy>('name');
   const sortByFunction = sortBy === 'name' ? sortByDisplayName : sortByIsFavorite;
   const [sortMenuVisible, setSortMenuVisible] = React.useState(false);
 
+  const toAccountDetail = (address: string) => {
+    navigation.navigate(myAccountScreen, {address});
+  };
+
+  const sortAccounts = (sort: SortBy) => {
+    setSortBy(sort);
+    setSortMenuVisible(false);
+  };
+
   return (
-    <Provider theme={theme}>
-      <SafeView edges={noTopEdges}>
-        {isLoading ? (
-          <LoadingView />
-        ) : (
+    <SafeView edges={noTopEdges}>
+      {isLoading ? (
+        <LoadingView />
+      ) : (
+        <>
+          <View style={styles.sortBy}>
+            <Menu
+              visible={sortMenuVisible}
+              onDismiss={() => {
+                setSortMenuVisible(false);
+              }}
+              anchor={
+                <TouchableOpacity
+                  onPress={() => {
+                    setSortMenuVisible(true);
+                  }}
+                  style={globalStyles.rowAlignCenter}>
+                  <Subheading>Sort by</Subheading>
+                  <Icon name="chevron-down" size={25} />
+                </TouchableOpacity>
+              }>
+              <Menu.Item
+                style={sortBy === 'name' ? {backgroundColor: colors.background} : undefined}
```

Comment:
NIT: another way to handle this is to disable the Item `<Menu.Item disabled={sortBy === 'name'} ... />`, this will change the color and also will prevent to call `sortAccounts` again